### ISSUE TITLE
Sims that need new screenshots

### ISSUE THREAD

**Issue by KatieWoe** [phetsims/QA#478]
Balloons and static Electricity, Molecules and Light, and Resistance in a Wire have accessibility features that are visible in their nav bar when you open the sim. However, they are not visible in the icons/screenshots. Should the screenshots for these sims be updated?

**Comment by ariel-phet** [comment]
@KatieWoe I don't think this is particularly high priority, the screenshot is more of a marker for people to recognize, and a way to get a sense of the simulation. We sometime have deviations due to maintenance releases and such. 

Molecules and light is scheduled for republication in the near future however, so you might want to make an issue in that simulation repo suggesting adding a new screenshot for the next publication, that would be an appropriate time (changing screenshots manually is a bit of a pain if I recall).

**Comment by KatieWoe** [comment]
Ok. Closing then.
